<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CBitmap Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#graphicsView {
	border: 1px solid grey;
}

QComboBox {
	padding-right: 0px; margin-right: 0px;
	padding-left: 2.5px;
}

QComboBox::down-arrow {
	image: none; width: 0; height: 0;
}

QComboBox::drop-down {
    width: 0; border: none; background: transparent;
	subcontrol-origin: padding;
    subcontrol-position: center right;
}

QComboBox QAbstractItemView {
    min-width: 120px;
    padding: 4px;
	text-align: left;
}

.QPushButton {
	padding: 2px;	
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyGraphicsArea" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform</set>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="viewportUpdateMode">
     <enum>QGraphicsView::FullViewportUpdate</enum>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbPen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pen</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/pen</normaloff>:/pen</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbEraser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Eraser</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/eraser</normaloff>:/eraser</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbImageInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Image Info &amp; Operations</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/info</normaloff>:/info</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbExport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Export to code</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/export</normaloff>:/export</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbResetView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reset View</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/home</normaloff>:/home</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbRedo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Redo</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/redo</normaloff>:/redo</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbUndo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Undo</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/undo</normaloff>:/undo</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxPenMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Shape</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbResolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>128*64</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbZoom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxFillMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fill Mode</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbCBitmapStatus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>560</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbImport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Import from code</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/import</normaloff>:/import</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="IndicatorButton" name="pbImportImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Import from Image</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uires.qrc">
      <normaloff>:/addimage</normaloff>:/addimage</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsArea</class>
   <extends>QGraphicsView</extends>
   <header>mygrar.h</header>
  </customwidget>
  <customwidget>
   <class>IndicatorButton</class>
   <extends>QPushButton</extends>
   <header>indicatorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="uires.qrc"/>
 </resources>
 <connections/>
</ui>
